server/mentionscrawler/blueprints/mention.py

```diff
@@ -41,3 +43,24 @@ def mention_response(user):
                          }
         output_mentions.append(output_mention)
     return jsonify(output_mentions), 200
+
+# Get details of a single mention
+@mention_bp.route("/mention", methods=["POST"])
+@enforce_json()
+@authenticate()
+def get_mention(user):
+    body = request.get_json()
+    if "id" not in body:
+        return bad_request_response("Invalid request! Missing fields!")
+
+    mention = Mention.query.filter_by(id=body.get("id")).first()
+    output_mention = {
+        URL_TAG: mention.url,
+        SITE_TAG: mention.site_id,
+        TITLE_TAG: mention.title,
+        SNIPPET_TAG: mention.snippet,
+        HITS_TAG: mention.hits
+    }
+    if mention is None:
+        return bad_request_response("There is no mention with that ID")
+    return jsonify(output_mention), 200
```

Comment:
I'd prefer it if we used the response functions I wrote for this, it ensures that responses are made in a uniform way and will be the same across all routes

Reply:
you mean the jsonify? I left that like that because the mentions route used the exact same return statement

Reply:
oh, my bad, I must have forgot to change it